docs/Overview.md

```diff
@@ -102,3 +102,4 @@ communication with the Klipper developers.
 - [Hall filament width sensor](Hall_Filament_Width_Sensor.md)
 - [Eddy Current Inductive probe](Eddy_Probe.md)
 - [Load Cells](Load_Cell.md)
+- [Load Cell Probes](Load_Cell_Probe.md)
```

Comment:
For what it is worth, it seems odd that we have two separate documents for load cells.  Seems like we could put it all into one document and I'd guess most users would prefer that.